packages/tabular/src/Happy/Tabular/Info.lhs

```diff
@@ -4,14 +4,13 @@ Generating info files.
 (c) 1993-2001 Andy Gill, Simon Marlow
 -----------------------------------------------------------------------------
 
-> module Info (genInfoFile) where
+> module Happy.Tabular.Info (genInfoFile) where
 
-> import Paths_happy            ( version )
-> import LALR                   ( Lr0Item(..), Goto(..), LRAction(..), ActionTable, GotoTable )
-> import GenUtils               ( str, interleave, interleave' )
 > import Data.Set ( Set )
 > import qualified Data.Set as Set hiding ( Set )
 > import Happy.Grammar
+> import Happy.Tabular.LALR   ( Lr0Item(..), LRAction(..), Goto(..), GotoTable, ActionTable )
+> import Paths_happy_tabular  ( version )
```

Comment:
I wonder if we should skip the

>   showHeader
>       = banner ("Info file generated by Happy Version " ++
>                 showVersion version ++ " from " ++ filename)

and keep that in the binary for now. I feel a bit weird embedding a library version number like that.